framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -480,13 +502,24 @@ public AnnotatedTypeFactory(BaseTypeChecker checker) {
                 case "jaifs":
                     wpiOutputFormat = WholeProgramInference.OutputFormat.JAIF;
                     break;
+                case "ajava":
+                    wpiOutputFormat = WholeProgramInference.OutputFormat.AJAVA;
+                    break;
                 default:
                     throw new UserError(
                             "Bad argument -Ainfer="
                                     + inferArg
                                     + " should be one of: -Ainfer=jaifs, -Ainfer=stubs");
             }
-            wholeProgramInference = new WholeProgramInferenceScenes(this);
+            if (checker.getOption("infer").equals("ajava")) {
```

Comment:
Would it be cleaner to check the variable `wpiOutputFormat` here rather than calling `getOption` and `String.equals`?